### ISSUE TITLE
react when user unauthorizes via OAuth provider

### ISSUE THREAD

**Issue by ChelseaStats** [gratipay/gratipay.com#823]
**Was:** option to delete account, change image and remove accounts when auth denied
1. added both twitter and github accounts, image persists with github image want twitter image.
2. removed github permissions from github, image and account persists on gittip.
3. no option to delete accounts.
4. no option to delete gittip account either.

thanks
##

**Comment by whit537** [comment]
Somehow Freshdesk is [reacting](https://gratipay.freshdesk.com/helpdesk/tickets/1753) to Facebook revocations:

![screen shot 2015-03-09 at 2 42 31 pm](https://cloud.githubusercontent.com/assets/134455/6561846/bab9c9dc-c66a-11e4-8e69-84a0e4e3ba38.png)

Reopening.

**Comment by rohitpaulk** [comment]
For facebook: 

![screenshot from 2015-03-10 00 21 34](https://cloud.githubusercontent.com/assets/3893573/6562026/81aa4c4c-c6bb-11e4-9c74-e1e090517dc2.png)

**Comment by whit537** [comment]
+1 https://hackerone.com/reports/129209

**Comment by sushil1208** [comment]
How can an attacker attack through the report  129209??

**Comment by whit537** [comment]
@sushil1208 Sorry, I don't understand. :-( Can you rephrase the question?

**Comment by whit537** [comment]
Closing in light of our decision to [shut down Gratipay](https://gratipay.news/the-end-cbfba8f50981).

Thank you all for a great run, and I'm sorry it didn't work out! 😞  💃
